man/org.freedesktop.systemd1.xml

```diff
@@ -3186,6 +3190,8 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
 
     <!--property PrivateMounts is not documented!-->
 
+    <!--property PrivateIPC is not documented!-->
```

Comment:
Please document these - I know we are bad and there's tons of undocumented ones, but we need to revert this trend

Reply:
Where to write the document?  I'm not sure.

Reply:
In this same file, a bit below - after ```<title>Methods</title>``` there's a series of ```<para>``` sections each documenting a method with ```<function>SomeMethod()</function>```

Reply:
There is a paragraph saying: "Most properties of the Service interface map directly to the corresponding settings in service unit files. For the sake of brevity, here's a list of all exceptions only". PrivateIPC and IPCNamespacePath are not exceptions so it seems not necessary to add a description.